app/bundles/LeadBundle/Entity/CustomFieldRepositoryTrait.php

```diff
@@ -361,6 +361,14 @@ protected function formatFieldValues($values, $byGroup = true, $object = 'lead')
         return $fieldValues;
     }
 
+    /**
+     * @return array<int, string>
+     */
+    public function getSearchableFieldAliases(string $object): array
```

Comment:
Can you please send the `LeadFieldRepository` as a param to this method as well? It's bad practice to use `$this` in traits as traits have no idea in which classes they are used. I know it's used in other methods but we should not extend this bad practice to new methods.

And when thinking about this. When you already will be passing the repository, you can already call the `getSearchableFieldAliases` method and avoid this method alltogether.

Reply:
Done.